processing/src/test/java/io/druid/segment/data/IncrementalIndexTest.java

```diff
@@ -208,6 +234,8 @@ public void testCaseSensitivity() throws Exception
   {
     long timestamp = System.currentTimeMillis();
     IncrementalIndex index = indexCreator.createIndex(defaultAggregatorFactories);
+    closer.register(index);
```

Comment:
also FYI you can inline the registration process, so you could just do 
`closer.register(indexCreator.createIndex(defaultAggregatorFactories))` if desired.